OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -62,10 +62,7 @@ public class HarvesterInfo : ITraitInfo, Requires<MobileInfo>
 		[Desc("Search radius (in cells) from the last harvest order location to find more resources.")]
 		public readonly int SearchFromOrderRadius = 12;
 
-		[Desc("Maximum duration of being idle before queueing a Wait activity.")]
-		public readonly int MaxIdleDuration = 25;
-
-		[Desc("Duration to wait before becoming idle again.")]
+		[Desc("Duration to wait before searching for resources again.")]
```

Comment:
How about "Interval to wait between searches when there are no resources nearby."?

Reply:
Done.